content/doc/find-help/troubleshooting.md

```diff
@@ -119,6 +119,42 @@ server.listen(8080);
 
 {{% /details %}}
 
+## Moderation
+
+{{% details title="What is moderation?" closed="true" %}}
+
+Moderation means your account is not accessible anymore. You might have broken some important rules of Clever Cloud's General Terms of Use.
+
+You are not allowed to create apps and addons, and run your current apps.
+
+{{% /details %}}
+
+{{% details title="Possible reasons why your app has been moderated" closed="true" %}}
+
+The main reasons why your account has been moderated are:
+- You run some forbidden apps such as those described on [this page](https://www.clever-cloud.com/acceptable-use-policy/)
+- You have not paid your pending invoices which are emitted since more than 30 days
+- The lack of personal information or the miss of payment information are considered as suscipicious by our system
+- Our payment platform has spotted you as emitting fraudulent payments.
+
+{{% /details %}}
+
+{{% details title="How to recover your application online?" closed="true" %}}
+
+Contact Clever Cloud's Support team at [support@clever-cloud.com](mailto:support@clever-cloud.com) to explain what your apps are designed for.
+
+Ensure all your pending invoices are paid.
+
+{{% /details %}}
+
+{{% details title="How to recover your data?" closed="true" %}}
+
+In case of a pending invoice aged more than 30 days, all your apps and addons might be deleted. However, we keep backups for 30 days after deletion.
```

Comment:
```suggestion
In case of a pending invoice aged more than 30 days, Clever Cloud might delete your apps and add-ons. However, we keep backups for 30 days after deletion.
```